apps/framework-cli/src/cli.rs

```diff
@@ -29,8 +29,9 @@ use crate::cli::routines::dev::{copy_old_schema, create_deno_files};
 use crate::cli::routines::flow::{create_flow_directory, create_flow_file};
 use crate::cli::routines::initialize::initialize_project;
 use crate::cli::routines::logs::{follow_logs, show_logs};
+use crate::cli::routines::migrate::generate_migration;
 use crate::cli::routines::templates;
-use crate::cli::routines::version::BumpVersion;
+use crate::cli::routines::version::bump_version;
```

Comment:
How did the JS casing get past the Clippy?

Reply:
It was a `struct` before now it is a function